### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by aberke**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -152,7 +152,7 @@ How does this risk increase when cohort IDs are combined with other forms of bro
 
 We analyze these questions using the preprocessed machine-weeks data.
 
-[notebooks/notebooks/unicity-analysis.ipynb](notebooks/notebooks/unicity-analysis.ipynb)
+[notebooks/notebooks/unicity-analysis.ipynb](notebooks/unicity-analysis.ipynb)
```
Comment (aberke) [inline README.md]: @dcalacci can you fix this double notebooks too?

**Comment by dcalacci** [comment]
This looks good to me overall! I fixed one small issue with a link in the readme. Those links should be changed once we merge into master so they no longer point to this branch. Some additional things that we could do to make it easier for anyone to follow along:

- [ ] number each notebook in the order they are run
- [ ] create a virtualenv / `requirements.txt` file with all the packages we use

I can't really tell if it's worth doing either of the above—it would probably take a couple of hours to create the requirements.txt and make sure all our code can be run with that environment. Do you think it's worth it?

File: README.md (hunk 1 above)
Comment (dcalacci) [reply README.md]: lol whoops

**Comment by aberke** [comment]
Thanks for the review! 
The links are relative links, so I don't think there should be an issue there.

For numbering the notebooks, was hoping the 'Analysis steps' heading would help make clear the order. There isn't a strict ordering since the steps for the different analyses are not necessarily linear.

I agree we should make a requirements.txt!

How about we merge for now and someone takes that on tomorrow?

**Comment by dcalacci** [comment]
Sounds good to me.

**Pull request merged**